content/blog/2016/2016-09-28-jenkins-world-2016-wrap-up-hub.adoc

```diff
@@ -0,0 +1,149 @@
+  Jenkins Area Meetup] and contributor to link:http://spring.io[Spring].
+* *link:https://github.com/jh86[Jon Hermansen]* - Organizer for the
+  link:http://www.meetup.com/Los-Angeles-Jenkins-Area-Meetup/[Los Angeles
+  Jenkins Area Meetup], developer and Pipeline user.
+* *link:https://github.com/omehegan[Owen Mehegan]* -
+  link:https://wiki.jenkins-ci.org/display/JENKINS/GitLab+Plugin[GitLab plugin]
+  contributor, release engineer and copy editor for jenkins.io.
+* *link:https://github.com/oleg-nenashev[Oleg Nenashev]* -
+  link:/projects/gsoc[Google Summer of Code] organizer, maintainer of multiple
+  plugins and link:http://www.meetup.com/St-Petersburg-Jenkins-Meetup/[St.
+  Petersburg Jenkins Area Meetup] organizer.
+* *link:https://github.com/bitwiseman[Liam Newman]* -
+  co-organizer link:http://www.meetup.com/Los-Angeles-Jenkins-Area-Meetup/[Seattle Jenkins Area Meetup],
+  contributor, and JobDSL and Pipeline user.
+* *link:https://github.com/orrc[Christopher Orr]* - Maintainer of multiple
+  Android-related plugins, including the
+  link:https://wiki.jenkins-ci.org/display/JENKINS/Android+Emulator+Plugin[Android Emulator plugin]
+  and contributor to numerous projects behind the scenes of Jenkins.
+* *link:https://github.com/cvega[Casey Vega]* - Organizer for the
+  link:http://www.meetup.com/Los-Angeles-Jenkins-Area-Meetup/[Los Angeles
+  Jenkins Area Meetup] and release engineer at Verizon Digital Media.
+* *link:https://github.com/markewaite[Mark Waite]* - Maintainer of the
+  link:http://wiki.jenkins-ci.org/display/JENKINS/Git%20Plugin[Git plugin] and
+  contributor to a number of other Git-related plugins.
+* *link:https://github.com/dty[Dean Yu]* - Long-time contributor, board member
+  and release engineer at Shutterfly.
+* *link:https://github.com/kzantow[Keith Zantow]* - contributor on the
+  link:https://wiki.jenkins-ci.org/display/JENKINS/Blue+Ocean+Plugin[Blue Ocean plugin]
+  and overall UX effort.
+
+By choice, I actually spent the majority of my time at the booth. It was
+fantastic to hear how people were using
+Jenkins and wanting to use Jenkins to do more. I answered dozens of questions
+on both days of the conference, often learning new things in the process of answering them.
+And for questions that were beyond any one person's knowledge, there was such a
+breadth of expertise, very few questions were beyond our combined abilities.
+
+While the Open Source Hub was generally very well attended, the lunch-time demos drew
+really big crowds.
+
+NOTE: a bit more text here about the demos in general.
+
+NOTE: still working on this list
+
+* link:https://github.com/kzantow[Keith Zantow] gave a live demo of
+  *link:/projects/blueocean[Blue Ocean] in Action*.
+* link:https://github.com/bitwiseman[Liam Newman] gave a live demo of
+  *Notifications with Jenkins Pipeline* based on his previous
+  link:/blog/2016/07/18/pipline-notifications/[blog post].
```

Comment:
IMO either drop "previous" from this sentence or lengthen to "based on his previous blog post on the subject"